lib/ain-ocean/src/api/rawtx.rs

```diff
@@ -1,27 +1,148 @@
-use std::sync::Arc;
+use std::{str::FromStr, sync::Arc};
 
+use ain_macros::ocean_endpoint;
 use axum::{
+    extract::{Json, Path},
     routing::{get, post},
-    Router,
+    Extension, Router,
 };
-use defichain_rpc::{Client, RpcApi};
-use super::path::Path;
+use bitcoin::Txid;
+use defichain_rpc::{json::Bip125Replaceable, RpcApi};
+use rust_decimal::{prelude::ToPrimitive, Decimal};
 
-async fn send_rawtx() -> String {
-    "Sending raw transaction".to_string()
+use super::{response::Response, AppContext};
+use crate::{
+    error::ApiError,
+    model::{
+        default_max_fee_rate, MempoolAcceptResult, RawTransaction, RawTxDto, TransctionDetails,
+        WalletTxInfo,
+    },
+    Error, Result,
+};
+
+#[ocean_endpoint]
+async fn send_rawtx(
+    Extension(ctx): Extension<Arc<AppContext>>,
+    Json(raw_tx_dto): Json<RawTxDto>,
+) -> Result<String> {
+    println!("{:?}", raw_tx_dto.hex.clone());
+    validate(raw_tx_dto.hex.clone())?;
+    let mut max_fee = Some(default_max_fee_rate().unwrap().to_sat());
+    if let Some(fee_rate) = raw_tx_dto.max_fee_rate {
+        let sat_per_bitcoin = Decimal::new(100_000_000, 0);
+        let fee_in_satoshis = fee_rate * sat_per_bitcoin;
+        max_fee = fee_in_satoshis.round().to_u64();
+    }
+    let trx = defichain_rpc::RawTx::raw_hex(raw_tx_dto.hex);
+    match ctx.client.send_raw_transaction(trx, max_fee).await {
```

Comment:
hmm.. is this needed? `defichain_rpc::RawTx::raw_hex()`??
```suggestion
    match ctx.client.send_raw_transaction(raw_tx_dto.hex, max_fee).await {
```